client/driver/raw_exec_test.go

```diff
@@ -261,6 +265,92 @@ func TestRawExecDriver_Start_Kill_Wait(t *testing.T) {
 	}
 }
 
+func TestRawExecDriver_Start_Kill_Wait_Cgroup(t *testing.T) {
```

Comment:
would add a comment that this test verifies that a raw exec task's process gets  killed correctly when configured to use cgroup.